etl/steps/data/garden/demography/2023-11-08/modal_age_death.meta.yml

```diff
@@ -0,0 +1,37 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Life Expectancy
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/dataset/
+dataset:
+  update_period_days: 365
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/tables/
+tables:
+  modal_age_death:
+    # Learn more about the available fields:
+    # http://docs.owid.io/projects/etl/architecture/metadata/reference/indicator/
+    variables:
+      modal_age_death:
+        title: Adult modal age at death
+        unit: years
+        description_short: |-
+          The most common age at death, among adults.
+
+        description_key:
+          - The adult modal age at death is the age (beyond infancy) at which the majority of deaths occur.
```

Comment:
Perhaps update this in line with the `The most common age at which people die in a given year.`, based on the feedback [here](https://owid.slack.com/archives/C04LNF8LBA8/p1699525970163009). Also maybe clarify exactly what infant means here? I think it is typically used to mean < 1 year old, but is it the same here, I see 10 being mentioned later.

Reply:
Thanks for the link. I've improved this section.